lib/Utils/Asset.php

```diff
@@ -17,21 +17,59 @@ class Asset
     protected static $assetManifest;
     protected static $loadFromCdn = false;
 
+    /**
+    * Gets the asset's url.
+    *
+    * @since 0.1.0
+    *
+    * @param string $asset The required asset.
+    *
+    * @return string/boolean Returns the url or false if the asset is not found.
+    */
     public static function requireUrl($asset)
     {
         return self::get('url', $asset);
     }
 
+    /**
+    * Gets the asset's path.
+    *
+    * @since 0.1.0
+    *
+    * @param string $asset The required asset.
+    *
+    * @return string/boolean Returns the path or false if the asset is not found.
+    */
     public static function requirePath($asset)
     {
         return self::get('path', $asset);
     }
 
+    /**
+    * Registers an asset.
+    * It can register a script or a style.
+    *
+    * @since 0.1.0
```

Comment:
oh yeah sorry, I added the CDN stuff after v0.1.0 ... that's not documented anywhere though, so maybe add `@since 0.2.0 support CDN parameter`?